pkg/pb/metainfo.proto

```diff
@@ -492,4 +493,9 @@ message SegmentDownloadResponse {
     repeated AddressedOrderLimit addressed_limits = 2;
     bytes                        encrypted_inline_data = 3;
     SegmentPosition              next = 4; // can be nil
+    bytes                        private_key = 5 [(gogoproto.customtype) = "PiecePrivateKey", (gogoproto.nullable) = false];
```

Comment:
Also here?

Reply:
for downloading its used for signing orders

Reply:
Yeah, forgot that it's not sent inside the order limit.

Reply:
Can we change the order of the fields so they make a little bit more sense?

Reply:
addressed_limits
private_key
encrypted_inline_data
next
??

Reply:
Yeah, something like that.